auth/auth.go

```diff
@@ -102,6 +101,7 @@ func NewClient(ctx context.Context, c *internal.AuthConfig) (*Client, error) {
 		}
 		email = svcAcct.ClientEmail
 	}
+
```

Comment:
may as well keep the spacing

Reply:
This is just a simple merge of the existing dev branch into master. We don't make manual changes here. These have to be addressed on dev as separate PRs.